builtin/providers/aws/resource_aws_vpc_dhcp_options.go

```diff
@@ -121,7 +121,7 @@ func resourceAwsVpcDhcpOptionsCreate(d *schema.ResourceData, meta interface{}) e
 	log.Printf("[DEBUG] Waiting for DHCP Options (%s) to become available", d.Id())
 	stateConf := &resource.StateChangeConf{
 		Pending: []string{"pending"},
-		Target:  "",
+		Target:  []string{},
```

Comment:
Hi clint.

Didn't see that during development.
The state resource tests passed for the empty case.

The code changed checks that length of target is greater than 0

On Monday 25 January 2016, Clint <notifications@github.com> wrote:

> In builtin/providers/aws/resource_aws_vpc_dhcp_options.go
> <https://github.com/hashicorp/terraform/pull/4767#discussion_r50738647>:
>
> > @@ -121,7 +121,7 @@ func resourceAwsVpcDhcpOptionsCreate(d *schema.ResourceData, meta interface{}) e
> >  	log.Printf("[DEBUG] Waiting for DHCP Options (%s) to become available", d.Id())
> >  	stateConf := &resource.StateChangeConf{
> >  		Pending: []string{"pending"},
> > -		Target:  "",
> > +		Target:  []string{},
>
> There are a handful of these Target: []string{} that I think have broken
> stuff (see https://travis-ci.org/hashicorp/terraform/jobs/104548430). I'm
> fixing, but was wondering if you hit this in development and possibly
> there's a bug elsewhere
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hashicorp/terraform/pull/4767/files#r50738647>.
>